config/locales/client.en.yml

```diff
@@ -53,6 +54,7 @@ en:
     motion_closed_by_expiry: "Proposal has closed"
     new_vote: "%{position} on the proposal"
     new_comment: "commented on the discussion"
+    comment_replied_to: "replied to your comment in the discussion"
```

Comment:
Please include the whole message in the translation string.. user names and discussion titles